### PR TITLE
Fix details pointed by Anthony

### PR DESCRIPTION
**Pull request by vitalii-bulyzhyn**
- Fix closing dialog
- Add reviewer's comment on the task status menu

### CODE REVIEW

File: shared/database/task/task_event.py [hunk 1]
```diff
@@ -51,6 +53,21 @@ def serialize(self):
...
+            TaskEvent.job_id == job_id,
+            TaskEvent.project_id == project_id,
+            TaskEvent.event_type == 'comment'
+        ).order_by(desc(TaskEvent.time_created)).first()
+
+        if (latest_comment == None):
+            return ""
+
+        comment_display = session.query(DiscussionComment).filter(DiscussionComment.id == latest_comment.comment_id).first()
+        return comment_display.__dict__["content"]
```
Comment (PJEstrada) [inline shared/database/task/task_event.py:69]: Correct me if I'm wrong, but I think here you con do just `comment_display.content` and you should get the same result maybe?

**Review by PJEstrada — APPROVED** [review]
Hey guys!

**Comment by anthony-sarkis** [comment]
Thanks Vitalii, And Thanks Pablo for reviewing :)

File: shared/database/task/task_event.py (hunk 1 above)
Comment (vitalii-bulyzhyn) [reply shared/database/task/task_event.py]: Yeah, you are totally right, just fixed this

**Pull request merged**